### PR TITLE
10 welcome page 4 less urgent help section

### PR DESCRIPTION
**Pull request by garussell**
### Description of changes

Added non-urgent checkboxes to the same form.  It was initially two separate boxes however this broke geolocation functioning and resulted in Capybara not recognizing error messages.  The solution is not perfect and deviates from our original wireframe (two different form boxes with two different colors).  If we have time, we could attempt a refactor that would require the following:

1.  Refactor JavaScript for geolocation to detect two different location field IDs.
2. Create a separate form for location that is utilized for two different forms/sections a) urgent and b) non-urgent.
3. Separate 'Get Help' buttons and 'Find Location' buttons for each form.

This was the most time-convenient and simple solution that resulted in all tests passing and all the functionality our users will need to be successful.

### Screenshots (if applicable) of relevant views
Screenshot 2023-09-16 at 1 33 05 PM

### Test Suite
- [x] Are all tests within the test suite passing?

### Specific Feedback Request(s)

If there is a way to separate the sections by color (while maintaining one JS file for geolocation and keeping the same form functions), that would be useful.

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -6,40 +6,66 @@
...
-<div class="select-urgent-services">
-  <h4>I need urgent help now!</h4>
-  
-  <%= form_with url: search_results_path, method: :get, local: true do |form| %>
-    <%= form.label :location, 'Enter your City, State, and/or Zip Code' %><br/>
-    <div class="display-flex">
-      <%= form.text_field :location, size: "40" %> 
+<div class="display-flex">
+  <div class="select-services">
+    <h4>I need urgent help now!</h4>
```
Comment (z-fitch) [inline app/views/welcome/index.html.erb:11]: Should the 'I need urgent help now' be below the location button? That way, it's clear that the location is for the entire form, not just the urgent help.

**Review by z-fitch — CHANGES_REQUESTED** [review]

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (garussell) [reply app/views/welcome/index.html.erb:11]: Yes, let me fix that.

Comment (garussell) [reply app/views/welcome/index.html.erb]: Done.

**Review by z-fitch — APPROVED** [review]
Thanks for making those changes! Looks good!

**Comment by z-fitch** [comment]
This PR is also in #12

**Pull request closed without merging**